MetaMask/metamask-extension
# 21142 improve sepolia goerli network icon implementation

## **Description**

**Problem**: The current implementation of Goerli and Sepolia network icons is inconsistent with current practices.

**Solution**: The test networks icons hard-coded in HTML were replaced with the preferred method of rendering icon images dynamically.

1. New image assets added for Sepolia and Goerli icons.
2. Links for new images added to app state data for test networks.
3. Deprecation message added to `getTestNetworkBackgroundColor` selector.

## **Manual testing steps**

1. Open the 'Select a network' menu in MM.
2. View the Goerli and Sepolia icons in the web inspector.
3. Observe the icons are an `<img>` with a `src` referencing their respective, newly added images.
4. Observe the icon letter is a larger size than the previous hard-coded implementation. 

### **Before**

<img width="598" alt="21142_before" src="https://github.com/MetaMask/metamask-extension/assets/4392888/a1227b62-f22c-4bce-9c68-e37379282a39">

### **After**

<img width="843" alt="21142_after" src="https://github.com/MetaMask/metamask-extension/assets/4392888/5b8806e6-9fe1-43c5-95e5-e2b2e5c661fc">

## **Related issues**

Fixes #21142

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
